specification/app/Microsoft.App/models/code-execution.tsp

```diff
@@ -36,25 +36,25 @@ union CodeInputType {
   string,
 
   @doc("Inline.")
-  inline: "inline",
+  inline: "Inline",
 
   @doc("InlineBase64.")
-  inlineBase64: "inlineBase64",
+  inlineBase64: "InlineBase64",
 
   @doc("InlineText.")
-  inlineText: "inlineText",
+  inlineText: "InlineText",
 }
 
 @doc("Execution Type")
 union ExecutionType {
   string,
 
   @doc("Synchronous")
-  synchronous: "synchronous",
+  synchronous: "Synchronous",
 }
 
 @doc("The properties of the code execution result.")
-model CodeExecutionResponseProperties {
+model SessionCodeExecutionResponseProperties {
```

Comment:
SessionCodeExecutionResponse can simply extend SessionCodeExecutionRequest with bunch of ready-only properties in response. and `executionTimeInMilliseconds`  & other properties under `diagnosticInfo` property bag